resqpy/fault/_grid_connection_set.py

```diff
@@ -1278,17 +1291,24 @@ def write_row(gcs, fp, name, i, j, k1, k2, axis, polarity):
             for feature_index in range(len(self.feature_list)):
                 feature_name = self.feature_list[feature_index][2].split()[0].upper()
                 cell_index_pairs, face_index_pairs = self.list_of_cell_face_pairs_for_feature_index(feature_index)
+                if tmult_array is not None:
+                    feature_mask = np.where(self.feature_indices == feature_index, 1, 0)
+                    feat_mult_array = np.extract(feature_mask, tmult_array)
+                else:
+                    feat_mult_array = np.ones(shape = (cell_index_pairs.shape[0],), dtype = float)
                 for side in sides:
-                    both = np.empty((cell_index_pairs.shape[0], 5), dtype = int)
+                    both = np.empty((cell_index_pairs.shape[0], 6), dtype = int)
```

Comment:
Let's add a trailing comment on the np.empty() line, indicating what the 6 elements will be, something like:
# axis, polarity, k, j, i, tmult

Reply:
Ha, I used a hash in my comment and it went into markdown mode!